docs/guidance/kubeflow-integration.md

```diff
@@ -0,0 +1,106 @@
+> Credit: This manifest refers a lot to the engineering blog ["Building a Machine Learning Platform with Kubeflow and Ray on Google Kubernetes Engine"](https://cloud.google.com/blog/products/ai-machine-learning/build-a-ml-platform-with-kubeflow-and-ray-on-gke) from Google Cloud.
+
+# Kubeflow: an interactive development solution
+
+# Requirements
+* Dependencies
+    * `kustomize`: v3.2.0 (Kubeflow manifest is sensitive to `kustomize` version.)
+    * `Kubernetes`: v1.23
+
+* Computing resources:
+    * 16GB RAM
+    * 8 CPUs
+
+# Example: Use Kubeflow to provide an interactive development envirzonment
+![image](../images/architecture.svg)
+
+## Step 1: Create a Kubernetes cluster with Kind.
+```sh
+# Kubeflow is sensitive to Kubernetes version and Kustomize version.
+kind create cluster --image=kindest/node:v1.23.0
+kustomize version --short
+# 3.2.0
+```
+
+## Step 2: Install Kubeflow v1.6-branch
+* This example installs Kubeflow with the [v1.6-branch](https://github.com/kubeflow/manifests/tree/v1.6-branch).
```

Comment:
Why do we need the 1.6 branch specifically? Would be good to add a brief comment

Reply:
Kubeflow is very sensitive to the Kubernetes version and the Kustomize version, so some instructions in this document may not work with other releases. In addition, some components will be renamed in future Kubeflow releases, and I am not familiar with the Kubeflow community's plan.